docs/src/integrations/iceberg.md

```diff
@@ -412,6 +409,181 @@ sequenceDiagram
+    participant Pipeline Tool
+    Orchestrator->>lakeFS: Create branch 'etl-2024-01-15'
+    Orchestrator->>lakeFS: Push tables to external catalog
+    lakeFS->>External Catalog: Register tables
+    Orchestrator->>Pipeline Tool: Run ETL pipeline
+    Pipeline Tool->>External Catalog: Read/write tables
+    Orchestrator->>lakeFS: Pull updated tables
+    lakeFS->>External Catalog: Read updated metadata
+    Orchestrator->>lakeFS: Merge 'etl-2024-01-15' into 'main'
+```
+
+### Configuration
+
+Remote catalogs are configured in your lakeFS configuration file.
+Each remote catalog requires a unique identifier and type-specific connection properties.
+
+#### AWS Glue Data Catalog
+
+Configure an AWS Glue catalog by specifying the region and AWS credentials:
+
+```yaml
+iceberg_catalog:
+  remotes:
+    - id: aws_glue_us_east_1
+      type: glue
+      glue:
+        region: us-east-1
+        access_key_id: <your-glue-key>
+        secret_access_key: <your-glue-secret>
+```
+
+#### Iceberg REST Catalog
+
+Configure a generic Iceberg REST catalog with basic authentication:
+
+```yaml
+iceberg_catalog:
+  remotes:
+    - id: remote_catalog
+      type: rest
+      rest:
+        uri: https://catalog.example.com/iceberg/api
+        credential: <client-id>:<client-secret>
+```
+
+Or with OAuth token authentication:
+
+```yaml
+iceberg_catalog:
+  remotes:
+    - id: remote_catalog
+      type: rest
+      rest:
+        uri: https://catalog.example.com/iceberg/api
+        oauth_token: <your-oauth-token>
+```
+
+### Push to remote
+
+Push operations register a table from lakeFS into a remote catalog.
+The table's metadata and data remain in lakeFS-managed storage, which are used as the pushed table's location.
+
+**API Endpoint**: `POST /iceberg/remotes/{catalog-id}/push`
+
+**Parameters**:
+- `source`: The lakeFS table location (repository, branch/reference, namespace, table name)
+- `destination`: The remote catalog location (namespace, table name)
+- `force_update`: (optional, default: `false`) Override the table if it already exists in the remote catalog
+- `create_namespace`: (optional, default: `false`) Create the namespace in the remote catalog if it doesn't exist
+
+**Example**:
+
+```bash
+curl -X POST "https://lakefs.example.com/iceberg/remotes/aws_glue_us_east_1/push" \
+  -H "Authorization: Bearer $LAKEFS_ACCESS_TOKEN" \
+  -H "Content-Type: application/json" \
+  -d '{
+    "source": {
+      "repository_id": "my-repo",
+      "reference_id": "main",
+      "namespace": ["default", "features"],
+      "table": "image_properties"
+    },
+    "destination": {
+      "namespace": ["main_features"],
+      "table": "image_properties"
+    },
+    "force_update": true,
+    "create_namespace": true
+  }'
+```
+
+This example pushes the table `my-repo.main.default.features.image_properties` from lakeFS to the AWS Glue catalog as `main_features.image_properties`,
+creating the namespace if needed and overwriting any existing table.
```

Comment:
perhaps add a small warning about force_update which can lead to lost writes on the remote